include/seqan3/core/detail/strong_type.hpp

```diff
@@ -112,11 +166,12 @@ namespace seqan3::detail
  *
  * \include test/snippet/core/detail/strong_type_adding_skills.cpp
  */
-template <typename value_t, typename derived_t, strong_type_skill skills = strong_type_skill::none>
+template <typename value_t, typename derived_t, strong_type_skill type_skills = strong_type_skill::none>
```

Comment:
```suggestion
template <typename value_t, typename derived_t, strong_type_skill skills_ = strong_type_skill::none>
```

After our new resolution :duck: and :runner:

Reply:
Hm I have the feeling this might be resolved in another commit but then again your commit history is clean?

Reply:
my history is always clean 🙆‍♂️ :)

Reply:
So you are just refusing the renaming? :D

Reply:
No, but I usually sleep during the night and obviously do not update the changes then. ;)